modules/prometheus-exporter_oracle/conf/17-dbvagent.yaml

```diff
@@ -0,0 +1,18 @@
+module: oracle
```

Comment:
Perhaps it is only my lack of knowledge about oracle but perhaps a tip could be useful for this detector.
For instance, here I cannot say if critical severity is relevant or not.

Reply:
it is related to a server process related to an oracle replication (standby db)
=> I will implement a tip. 
SOS DBA are aware about this, and will be fully able to manage the incidents related to.